### PR TITLE
Create 0xE4F6D46C244Bb7Cf3e218CDFB5C35cf9a4d9c920.json

### PR DESCRIPTION
**Pull request by projectdonkey**
Add DONK token

### CODE REVIEW

**Review by ligi — CHANGES_REQUESTED** [review]
Thanks! But to merge it the address must be in [ERC-55](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-55.md) (filename and address field)

It actually seems good in the filename - but for sure not in the field.

You can create addresses with checksum e.g. with this tool: https://ethsum.netlify.app/

**Comment by projectdonkey** [comment]
Hi 

The requested changed has been completed thanks

File: tokens/eth/0xE4F6D46C244Bb7Cf3e218CDFB5C35cf9a4d9c920.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  "symbol": "DONK",
+  "address": "0xE4F6D46C244Bb7Cf3e218CDFB5C35cf9a4d9c920.json",
```
Comment (ligi) [inline tokens/eth/0xE4F6D46C244Bb7Cf3e218CDFB5C35cf9a4d9c920.json:3]:
```suggestion
  "address": "0xE4F6D46C244Bb7Cf3e218CDFB5C35cf9a4d9c920",
```

**Review by ligi — APPROVED** [review]

**Comment by projectdonkey** [comment]
Please list Donkey

**Pull request merged**